src/components/OTP.vue

```diff
@@ -145,6 +145,11 @@ export default {
   },
 
   computed: {
+    /** Retrieve phone number stored in vuex */
+    ...mapState({
+      statePhoneNumber: (state) => state.phoneNumber,
+    }),
```

Comment:
```suggestion
    ...mapState({
      /** Retrieve phone number stored in vuex */
      statePhoneNumber: (state) => state.phoneNumber,
    }),
```
I think it would make sense to move the comment inside, just above a particular state value that you're mapping. Because when more state variables will come, then you might want to add a comment for each of them